app/front/scripts/services/downloader.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
But why do we need to proxy all requests through another server? Why not go directly to the API server?